doc/source/whatsnew/v0.21.0.txt

```diff
@@ -144,6 +144,7 @@ Bug Fixes
 ~~~~~~~~~
 
 - Fixes regression in 0.20, :func:`Series.aggregate` and :func:`DataFrame.aggregate` allow dictionaries as return values again (:issue:`16741`)
+- Fixes `__setitem__` with a tuple for dataframes with a tz-aware `DatetimeIndex` (:issue: `16889`)
```

Comment:
this is not correct:

``DataFrame.loc`` for setting wth alignment and a tz-aware DatetimeIndex (use double-back ticks around DTI)